csrankings-9.csv

```diff
@@ -2030,6 +2030,7 @@ Zhenjiang Li,City University of Hong Kong,https://www.cs.cityu.edu.hk/~zhenjili/
 Zhenkai Liang,National University of Singapore,https://www.comp.nus.edu.sg/~liangzk,SXfOJjcAAAAJ
 Zhenlin Wang,Michigan Technological University,https://www.mtu.edu/cs/department/faculty-staff/faculty/z-wang,ZAtffHkAAAAJ
 Zhenman Fang,Simon Fraser University,http://www.sfu.ca/~zhenman,v_FAgbgAAAAJ
+Zhengming (Allan) Ding,Tulane Universiry,https://sse.tulane.edu/cs/faculty/Ding,TKbyRRsAAAAJ,

```

Comment:
Doesn't match DBLP name (https://dblp.org/pid/122/3547.html), has a spurious trailing comma, and "University" is misspelled.